Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -527,27 +527,6 @@ theorem norm_nonneg' (a : E) : 0 ≤ ‖a‖ := by
 theorem abs_norm' (z : E) : |‖z‖| = ‖z‖ := abs_of_nonneg <| norm_nonneg' _
 #align abs_norm abs_norm
 
-namespace Mathlib.Meta.Positivity
-
-open Lean Meta Qq Function
-
-/-- Extension for the `positivity` tactic: multiplicative norms are nonnegative, via
-`norm_nonneg'`. -/
-@[positivity Norm.norm _]
-def evalMulNorm : PositivityExt where eval {_ _} _zα _pα e := do
-  let .app _ a ← whnfR e | throwError "not ‖ · ‖"
-  let p ← mkAppM ``norm_nonneg' #[a]
-  pure (.nonnegative p)
```

Comment:
This will conflict with #10196, but not in a major way

Reply:
Happy to fix it when we merge that:)